src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -551,7 +550,9 @@ const DateTimePicker = ({
   useEffect(
     () => {
       if (defaultValue || humanValue === null) {
-        parseDefaultValue();
+        parseDefaultValue(defaultValue);
+        setLastAppliedValue(defaultValue);
+        setIsExpanded(false);
```

Comment:
Why should the menu close when the defaultValue changes? What if the `expanded` prop is passed in as true?

Reply:
Removed setIsExpanded(false);